src/templates/partials/header.hbs

```diff
@@ -21,7 +21,10 @@
     </div>
   </menu>
 
-  <h1><a href="/" title="Link to home page">pieshop</a></h1>
+  <h1 id="logo"><a href="/" title="Link to home page"><svg viewBox="0 0 24 24">
```

Comment:
The logo is causing the title to conflict with the search box on smaller screens.

---

`master`
![screenshot 2018-06-29 at 16 14 51](https://user-images.githubusercontent.com/108052/42100403-fc9bad74-7bb7-11e8-8650-67b0448be9bb.png)

---

`logo`
![screenshot 2018-06-29 at 16 15 42](https://user-images.githubusercontent.com/108052/42100433-0fe1adc0-7bb8-11e8-9a38-a33b6f3e73c9.png)

Reply:
We can fix this, but we should avoid displaying both the logo and name at smaller viewport sizes.